tests/mochs/MochComponent.jsx

```diff
@@ -25,7 +26,7 @@ class MochComponent extends React.Component {
 }
 
 MochComponent.propTypes = {
-	title: React.PropTypes.string,
+	title: PropTypes.string,
 };
 
 export default MochComponent;
```

Comment:
Is there a reason this is called `MochComponent` instead of `MockComponent`? Or is it just a typo?